addons/stock/models/stock_orderpoint.py

```diff
@@ -686,10 +686,10 @@ def _procure_orderpoint_confirm(self, use_new_cursor=False, company_id=None, rai
 
                 # Log an activity on product template for failed orderpoints.
                 for orderpoint, error_msg in all_orderpoints_exceptions:
-                    existing_activity = self.env['mail.activity'].search([
+                    existing_activity = self.env['mail.activity'].search_count([
                         ('res_id', '=', orderpoint.product_id.product_tmpl_id.id),
                         ('res_model_id', '=', self.env.ref('product.model_product_template').id),
-                        ('note', '=', error_msg)])
+                        ('note', '=', html_sanitize(error_msg))], limit=1)
```

Comment:
```suggestion
                        ('note', 'like', error_msg)], limit=1)
```
I don't think we need to bother that far. Good call on the `search_count` though 👍